fix: Adding batch fetch and insert to migration guide. (#296)
googleapis/langchain-google-alloydb-pg-python: samples/migrations/migrate_vectorstore_to_alloydb.md

        ```
        To know more about working with multiple namespaces, see [docs](https://docs.pinecone.io/guides/indexes/use-namespaces).

>    5. (Optional) For large datasets we can return an iterator

Reviewer: ```suggestion
   5. (Optional) For large datasets we can fetch data in batches
```

>    5. (Optional) For large datasets we can return an iterator

Reviewer 2: This will not be optional. This should overwrite the original code snippets

        ```python

>         def get_all_pinecone_ids(index, batch_size):

Reviewer 2: Please add type hints

    )
    ```

4. (Optional) To add in large data from the iterator

    ```python
    iterator = get_all_pinecone_data(pinecone_index, batch_size=5)
    documents_length = 0

>     for ids, embeddings, content, metadatas  in iterator:

Reviewer 2: This can also be improved through batching the insertion like https://github.com/googleapis/langchain-google-alloydb-pg-python/blob/779f18a4d5e749e739ec6adb8e736c93e5d9d90d/src/langchain_google_alloydb_pg/utils/pgvector_migrator.py#L92

>       documents_length = documents_length + len(ids)

Reviewer 2: ```suggestion
      documents_length += len(ids)
```
